<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>声音交互软件</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,15" columnstretch="0,0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinAndMaxSize</enum>
    </property>
    <property name="leftMargin">
     <number>9</number>
    </property>
    <item row="2" column="0">
     <widget class="Line" name="line_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <layout class="QVBoxLayout" name="layout_graph">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <layout class="QHBoxLayout" name="layout_output_device" stretch="1,3">
      <item>
       <widget class="QLabel" name="output_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>输出设备：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="output_device"/>
      </item>
     </layout>
    </item>
    <item row="2" column="4">
     <widget class="Line" name="line_11">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QHBoxLayout" name="layout_button">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="button_play">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_stop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>暂停</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_record">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>采集</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_end">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>终止</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QHBoxLayout" name="layout_input_device" stretch="1,3">
      <item>
       <widget class="QLabel" name="input_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>输入设备：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="input_device">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_close"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_generate">
    <property name="title">
     <string>生成</string>
    </property>
    <widget class="QMenu" name="menu_modify">
     <property name="title">
      <string>调制信号</string>
     </property>
     <addaction name="action_AM"/>
     <addaction name="action_FM"/>
    </widget>
    <addaction name="action_single"/>
    <addaction name="menu_modify"/>
    <addaction name="action_sweep"/>
   </widget>
   <widget class="QMenu" name="menu_analyze">
    <property name="title">
     <string>分析</string>
    </property>
    <addaction name="action_freq"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_generate"/>
   <addaction name="menu_analyze"/>
  </widget>
  <action name="action_new">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_single">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>单频信号</string>
   </property>
  </action>
  <action name="action_sweep">
   <property name="text">
    <string>扫频信号</string>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_AM">
   <property name="text">
    <string>AM</string>
   </property>
  </action>
  <action name="action_FM">
   <property name="text">
    <string>FM</string>
   </property>
  </action>
  <action name="actionxianx">
   <property name="text">
    <string>xianx</string>
   </property>
  </action>
  <action name="action_freq">
   <property name="text">
    <string>频域分析</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>button_play</tabstop>
  <tabstop>button_stop</tabstop>
  <tabstop>button_record</tabstop>
  <tabstop>button_end</tabstop>
  <tabstop>input_device</tabstop>
  <tabstop>output_device</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
